### DISCUSSION TITLE
Mirroring labeled images

### DISCUSSION THREAD

**Discussion by matroskin313** [AlexeyAB/darknet#7585] [Q&A]
Hi,
I need to mirror horizontally already labeled photos in Labelimg with annotation files to use it in YOLOv4 darknet.
So I could icrease the number of photos and objects.
Could you give me any idea how I could make it fast?
Is there any apps or one way is to write the code?

Thank you for help.

**Comment by stephanecharette** [comment]
See:  https://www.ccoderun.ca/darkmark/DataAugmentationMisc.html#Flip

**Reply by matroskin313** [reply]
thank you

**Reply by matroskin313** [reply]
Hello,
Don't you know, does this flip=1 exist in cfg file by default?
I added it, but mAP is the same
